pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -24,6 +24,7 @@ from pandas._libs.tslibs.conversion cimport (
 from pandas._libs.tslibs.nattype cimport NPY_NAT, c_NaT as NaT
 from pandas._libs.tslibs.np_datetime cimport (
     check_dts_bounds, npy_datetimestruct, dt64_to_dtstruct)
+from pandas._libs.tslibs.np_datetime import OutOfBoundsDatetime
```

Comment:
is this used here?

Reply:
Nope, removed.

I'm holding out hope that one day we'll be able to lint these files.